fix(telegram): stop measuring http://web.telegram.org/ (#269)
ooni/spec: nettests/ts-020-telegram.md

* 149.154.167.91
* 149.154.171.5

The test establishes TCP connection to all the access point IP addresses and

> then attempts to issue a POST request to each of them.

Author: ```suggestion
then attempts to issue a POST HTTP request to each of them.
```

If all TCP connections on ports 80 and 443 to Telegram’s access point IPs fail
we consider Telegram to be blocked. The key `telegram_tcp_blocking` is used
to indicate if we believe telegram to be blocked at the TCP level.

If at least an HTTP request returns back a response, we consider Telegram to

> not be blocked. The key `telegram_http_blocking` is used to indicate if we believe

Author: ```suggestion
be working as intended. The key `telegram_http_blocking` is used to indicate if we believe
```

> telegram to be blocked at the HTTP level.

Author: ```suggestion
telegram DCs are blocked at the HTTP level.
```
